lib/resources/docker_container.rb

```diff
@@ -78,7 +78,7 @@ def repo
     end
 
     def tag
-      image.split(':')[1] unless image.nil?
+      image.split(':')[-1] unless image.nil? || !image.index(':')
```

Comment:
@jerryaldrichiii added the suggestion to change this from `index` to `include?` :)